src/string.test.ts

```diff
@@ -0,0 +1,12 @@
+import { words } from './string';
+
+describe('words', () => {
+  it.each([
+    ['fooBar', ['foo', 'Bar']],
+    ['', []],
+    ['a', ['a']],
+    ['AI', ['AI']],
```

Comment:
Currently failing, the function returns `['A', 'I']`.

Reply:
I actually found a use case where this might be important for us ['testDNAAndRNAMix', ["test","DNA","And","RNA","Mix"]] and lodash would solve this correctly. Honestly, I do not know how to solve this in an efficient way with and simple regex and would fall back to the lodash solution...

Reply:
I see value in keeping the dependencies for this project low, but guess I am fine with including lodash. Lets us drop the implementation for `words` and `capitalizeWord` 👍

Reply:
ok, this means that I will close the PR and we will solve this directly in the parent project with lodash